JASP-Engine/JASP/R/principalcomponentanalysis.R

```diff
@@ -842,44 +872,59 @@ PrincipalComponentAnalysis <- function(dataset = NULL, options, perform = "run",
 
 ### Screeplot:
 
-.screePlot <- function(dataset, options, perform) {
+.screePlot <- function(dataset, options, perform, oldPlot = NULL, plotType) {
+
```

Comment:
@hummyhummy in a common r file

Reply:
this is a common function only in file principalcomponentanalysis.R